dist/encoder.js

```diff
@@ -34,13 +34,40 @@ var Encoder = function () {
   }
 
   _createClass(Encoder, null, [{
-    key: 'getFunctionHash',
+    key: 'getEventHash',
+
+    /*
+     * Converts an event object from the ABI to its hash format.
+     * @param eventObj The json object of the event taken from the ABI.
+     * @return The event hash.
+     */
+    value: function getEventHash(eventObj) {
```

Comment:
loos like 1getEventHash()1 is highly similar to 1getFunctionHash()1 except the output byte length, can we build 1getFunctionHash()` using `getEventHash()`?

Reply:
good call, fixed!